Source/VCGeneration/VCGen.cs

```diff
@@ -749,9 +758,22 @@ private void ConvertCFG2DAGStandard(Implementation impl, Dictionary<Block, List<
                 b = new Bpl.LoopInvMaintainedAssertCmd(c.tok, c.Expr, c);
               }
 
-              b.Attributes = c.Attributes;
+              b.Attributes = (QKeyValue)c.Attributes?.Clone();
```

Comment:
Could you improve the names of `b` and `c` and stop them from being assigned multiple times?